docs/src/01-installation.md

```diff
@@ -50,7 +50,7 @@ You can install the ZKsync Solidity compiler toolchain using the following metho
 
 ## Installing *solc*
 
-Running *zksolc* requires the [fork of the Solidity compiler *solc*](https://github.com/matter-labs/era-solidity/releases), which is called by *zksolc* as a child process. To point *zksolc* to the location of *solc*, use one of the following methods:
+Running *zksolc* requires the [fork of the Solidity compiler *solc*](https://github.com/matter-labs/era-solidity/releases) where [we fixed several issues](./eravm/02-evm-assembly-translator.md#zksync-fork-of-solc) with lowering of EVM assembly to LLVM IR. The fork is called by *zksolc* as a child process. To point *zksolc* to the location of *solc*, use one of the following methods:
```

Comment:
General comment: I'd recommend adding some kind of MD linter which would limit the line length.

Reply:
I prefer word wrap to artificial line breaks, because line breaks must be always adjusted when you insert something in the middle of a line.
Let me know if you're aware of any issues with my approach.

Reply:
Up to you; limited line length makes reviewing easier (subjectively)